src/main/resources/hudson/tasks/junit/JUnitResultArchiver/config.jelly

```diff
@@ -33,6 +33,9 @@ THE SOFTWARE.
 	<f:entry field="keepLongStdio" title="">
 		<f:checkbox name="keepLongStdio" title="${%Retain long standard output/error}"/>
 	</f:entry>
+	<f:entry title="${%Suite name prefix}" field="suitePrefix">
```

Comment:
There is no advanced section, so I chose to have it open instead of introducing one.